src/plugins/ComputedPlugin.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+/**
+ * A plugin for a computed
+ */
+function ComputedPlugin() {
+  const name = 'computed';
+
+  /**
+   * Augment the base schema with the computed type
+   * @param {*} schema
+   */
+  const augmentSchema = ((schema) => {
+    const additions = {
+      inlines: {
+        computed: {
+          nodes: [{
+            match: { object: 'text' }
+          }]
+        },
+      },
+    };
+
+    const newSchema = JSON.parse(JSON.stringify(schema));
+    newSchema.inlines = { ...newSchema.inlines, ...additions.inlines };
+    newSchema.document.nodes[0].match.push({ type: 'computed' });
+    newSchema.blocks.paragraph.nodes[0].match.push({ type: 'computed' });
+    return newSchema;
+  });
+
+  /**
+   * Do not allow computed fields to be edited
+   *
+   * @param {*} value - the Slate value
+   * @param {string} code - the key code
+   */
+  const isEditable = ((value, code) => {
+    const inComputed = value.inlines.size > 0 && value.inlines.every(node => node.type === 'computed');
+    if (inComputed) {
+      return false;
```

Comment:
I *think* this is saying that if *every* selected inline is a `computed` then we are not editable. Should it be if *any* selected node is a `computed` then we are not editable. Try testing with a mixed highlight that includes both a `variable` inline and a `computed` inline and see if you can edit.

Reply:
good catch. this should use `.some`